src/lib/Repository/UiUserService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\UserService;
+
+/**
+ * Service for loading user.
+ */
+class UiUserService
+{
+    /**
+     * @var UserService
+     */
+    private $userService;
+
+    /**
+     * @var Repository
+     */
+    private $repository;
+
+    public function __construct(Repository $repository, UserService $userService)
+    {
+        $this->repository = $repository;
+        $this->userService = $userService;
+    }
+
+    /**
+     * Loads a user ignoring privileges.
+     *
+     * @param mixed $userId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    public function loadUserIgnoringPermissions($userId)
```

Comment:
I have been bothered by `loadUser` returning `null`, but I couldn't figure out why.

In the Repository's API, `loadAnything()` _will_ throw a NotFound. It's on purpose. You're asking to _load_ something that can't be found. We ended up preferring `findStuff` when there might not be a result.

What about `findUser()` ? Or `findUserById()`.

Not a request for change for this PR though. Up to you, if you see a way to implement this that fits.

The `IgnoringPermissions` part leaves me a bit skeptical as well. The fact is that for what we're doing here, we always want to ignore permissions. We want those users names and links. Could we make that the default for this service ? We may have to name it a bit differently to reflect that, but if we consider that those services reflect the business logic of the frontend, it can make sense.